### PR TITLE
Dashboard tables

### PR DESCRIPTION
**Pull request by LukeHouge**

### CODE REVIEW

**Comment by EUdds** [comment]
@LukeHouge A place to display the age of the last packet is still needed. Could you please place this where you see fit. `<div id="age"></div>`

**Comment by EUdds** [comment]
@LukeHouge Good progress, can you remove the front-end rng, and set functions to color the table indicator, according to the numbers coming in.

**Comment by EUdds** [comment]
Also note... [https://i.imgur.com/HRQP1dx.png](url)

**Comment by EUdds** [comment]
Also note https://i.imgur.com/HRQP1dx.png

**Review by EUdds — CHANGES_REQUESTED** [review]
See comments

**Comment by LukeHouge** [comment]
> @LukeHouge A place to display the age of the last packet is still needed. Could you please place this where you see fit. `<div id="age"></div>`

done, also added logos and connection status for links (like in state machine last year)

File: dashboard/public/javascripts/dynamicStyling.js [hunk 1]
```diff
@@ -1,135 +1,135 @@
...
-        x[2].style.backgroundColor = '#FC6962';
-        w++;
-      }
-      else if (y>90) {
-        x[2].style.backgroundColor = '#C8EEC4';
-      }
-      else {
-        x[2].style.backgroundColor = '#fff';
-      }
+  var w=0;
```
Comment (EUdds) [inline dashboard/public/javascripts/dynamicStyling.js:3]: Can you write some pseudocode for this file, I'm having a hard time following what is going on.

Comment (EUdds) [reply dashboard/public/javascripts/dynamicStyling.js:3]: I get a constant repeating error from this file as well
https://i.imgur.com/HRQP1dx.png
Please look into this

Comment (LukeHouge) [reply dashboard/public/javascripts/dynamicStyling.js:3]: Fixed the errors, most caused by undefined variables after the random generator was taken out. I put in empty var's right now, but just need to be changed to represent the incoming data. I will add comments to that file as well.

Comment (LukeHouge) [reply dashboard/public/javascripts/dynamicStyling.js:3]: Comments added to that file as well.

**Review by EUdds — APPROVED** [review]

**Pull request merged**